vulnerabilities/tests/test_archlinux.py

```diff
@@ -100,3 +101,64 @@ def assert_for_package(self, name, version, cve_ids=None):
 
         if cve_ids:
             assert cve_ids == {v.vulnerability_id for v in qs[0].vulnerabilities.all()}
+
+
+def test_parse_advisory_single():
```

Comment:
Please check `test_debian.py` , we need to assert the data inside AdvsioryData object